website/docs/user-guides/leave-project.md

```diff
@@ -4,4 +4,23 @@ title: Leave a Project
 sidebar_label: Leave Project
 ---
 
-## Coming soon
+## Introduction
+
+You can leave a project that you no longer wish to be a part of.
+
```

Comment:
Can you add `Before you begin` for this user guide too!

Reply:
`change-project-name` and `leave-project` are under project management in the sidebar which is the same logical section.. this is why added it in the first section of `change-project-name` only.